examples/packages/ssr/src/app.js

```diff
@@ -11,21 +12,33 @@ const Header = () => (
       height: 56,
       display: 'flex',
       alignItems: 'center',
+      '@media only screen and (max-width:1100px)': {
+        background: 'green',
+      },
     }}>
     server side rendering
   </header>
 );
 
+const Button = ({ count, setCount }) => (
+  <button css={{ background: 'white', borderRadius: 3 }} onClick={() => setCount((count += 1))}>
+    Count {count}
+  </button>
+);
+
 const Content = () => (
   <main css={{ padding: 8 * 4, height: '200vh', background: 'blue' }}>content</main>
 );
 
 export default function Home() {
+  const [count, setCount] = React.useState(0);
   return (
     <div css={{ fontSize: '100%' }}>
       <Header />
+      <Button count={count} setCount={setCount} />
       <Content />
       <Footer />
+      <SSRCacheComponent />
```

Comment:
Not sure how we want to handle this but I put this here for testing but thinking about it I think this should go in the ```server.js``` file and wrapped in the ```CacheProvider```.

Like: 
```js
... snip ...
import { SSRCacheComponent, CC as CacheProvider } from '@compiled/react/runtime'; // perhaps we can re-export CC as CacheProvider so you can just import CacheProvider directly?
import App from './app';

// eslint-disable-next-line @typescript-eslint/no-var-requires
const assets = require(process.env.RAZZLE_ASSETS_MANIFEST);

const server = express();
server
  .disable('x-powered-by')
  .use(express.static(process.env.RAZZLE_PUBLIC_DIR))
  .get('/*', (req, res) => {
    const context = {};
    const markup = renderToString(
      <CacheProvider>
        <App />
        <SSRCacheComponent />
      </CacheProvider>
    );
... snip ...